S3 error on local analysis run during results re-import

Running a local analysis job fails on the Shapefile results re-import step with an S3 error.

<Details>
<AnalysisJob: EXPORTING Dunwoody>: Loaded overall_scores from /data/output/2abefe94-068b-4b0a-a7f6-8970280154a1/neighborhood_overall_scores.csv
[2018-12-27 15:59:34 +0000] [940] [INFO] Running import for analysis job 2abefe94-068b-4b0a-a7f6-8970280154a1.
[2018-12-27 15:59:34 +0000] [940] [INFO] Importing results back from shapefile: results/2abefe94-068b-4b0a-a7f6-8970280154a1/neighborhood_census_blocks.zip
[2018-12-27 15:59:34 +0000] [940] [ERROR] Error importing <class 'pfb_analysis.models.CensusBlocksResults'> shapefile for job: 2abefe94-068b-4b0a-a7f6-8970280154a1
Traceback (most recent call last):
  File "/opt/pfb/django/pfb_analysis/management/commands/import_results_shapefiles.py", line 64, in import_shapefile
    import_tmpdir = geom_from_results_url(s3_job_url(job, shpfile_name))
  File "/opt/pfb/django/pfb_analysis/management/commands/import_results_shapefiles.py", line 48, in geom_from_results_url
    local_zipfile)
  File "/usr/local/lib/python2.7/dist-packages/boto3/s3/inject.py", line 130, in download_file
    extra_args=ExtraArgs, callback=Callback)
  File "/usr/local/lib/python2.7/dist-packages/boto3/s3/transfer.py", line 307, in download_file
    future.result()
  File "/usr/local/lib/python2.7/dist-packages/s3transfer/futures.py", line 73, in result
    return self._coordinator.result()
  File "/usr/local/lib/python2.7/dist-packages/s3transfer/futures.py", line 233, in result
    raise self._exception
ClientError: An error occurred (404) when calling the HeadObject operation: Not Found
[2018-12-27 15:59:34 +0000] [940] [INFO] Importing results back from shapefile: results/2abefe94-068b-4b0a-a7f6-8970280154a1/neighborhood_ways.zip
[2018-12-27 15:59:34 +0000] [940] [ERROR] Error importing <class 'pfb_analysis.models.NeighborhoodWaysResults'> shapefile for job: 2abefe94-068b-4b0a-a7f6-8970280154a1
Traceback (most recent call last):
  File "/opt/pfb/django/pfb_analysis/management/commands/import_results_shapefiles.py", line 64, in import_shapefile
    import_tmpdir = geom_from_results_url(s3_job_url(job, shpfile_name))
  File "/opt/pfb/django/pfb_analysis/management/commands/import_results_shapefiles.py", line 48, in geom_from_results_url
    local_zipfile)
  File "/usr/local/lib/python2.7/dist-packages/boto3/s3/inject.py", line 130, in download_file
    extra_args=ExtraArgs, callback=Callback)
  File "/usr/local/lib/python2.7/dist-packages/boto3/s3/transfer.py", line 307, in download_file
    future.result()
  File "/usr/local/lib/python2.7/dist-packages/s3transfer/futures.py", line 73, in result
    return self._coordinator.result()
  File "/usr/local/lib/python2.7/dist-packages/s3transfer/futures.py", line 233, in result
    raise self._exception
ClientError: An error occurred (404) when calling the HeadObject operation: Not Found
Updating job status: EXPORTING Uploading results
</Details>


The results re-import then succeeds on either re-running the `export_connectivity.sh` script directly within the analysis container after the analysis run exits, or by re-running the entire analysis job without clearing out the existing results.

Connects #607

--- a/src/analysis/scripts/export_connectivity.sh
+++ b/src/analysis/scripts/export_connectivity.sh
@@ -175,8 +175,6 @@ then
         ec_export_table_csv "${OUTPUT_DIR}" "neighborhood_overall_scores"
         # Send overall_scores to Django app
         update_overall_scores "${OUTPUT_DIR}/neighborhood_overall_scores.csv"
-        # Insert shapefile geometries into Django app DB if we have PFB_JOB_ID
-        PFB_S3_STORAGE_BUCKET="${AWS_STORAGE_BUCKET_NAME}" import_geometries_for_job
 
         if [ -n "${AWS_STORAGE_BUCKET_NAME}" ] && [ -n "${PFB_S3_RESULTS_PATH}" ]
         then
@@ -185,5 +183,8 @@ then
           aws s3 cp --quiet --recursive "${OUTPUT_DIR}" \
             "s3://${AWS_STORAGE_BUCKET_NAME}/${PFB_S3_RESULTS_PATH}"
         fi
+
+        # Insert shapefile geometries into Django app DB if we have PFB_JOB_ID
+        PFB_S3_STORAGE_BUCKET="${AWS_STORAGE_BUCKET_NAME}" import_geometries_for_job
     fi
 fi

